<!DOCTYPE html>
<html lang="en">

<head>
    <link rel="icon" href="labsymb.jpg" type="image/x-icon">
    <meta http-equiv="refresh" content="300">
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
	
	
     <title>Amjad Khan - Research</title>

    <!-- Bootstrap Core CSS -->
    <link href="css/bootstrap.css" rel="stylesheet">

    <!-- Custom CSS -->
    <link href="css/dme65.css" rel="stylesheet">

    <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
        jax: ["input/TeX", "output/HTML-CSS"],
        tex2jax: {
            inlineMath: [['$', '$']],
            displayMath: [['$$', '$$']],
            processEscapes: true,
            skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
        },
        messageStyle: "none",
        "HTML-CSS": { preferredFont: "TeX", availableFonts: ["STIX","TeX"] }
    });
    </script>
    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_CHTML"
    type="text/javascript"></script>
</head>

<body>
  <!-- Navigation -->
	
	<!--<nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">-->
	<nav class="navbar navbar-default navbar-fixed-top" role="navigation">
    <div class="container black">
      <!-- Brand and toggle get grouped for better mobile display -->
      <div class="navbar-header">
        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar">
          <span class="sr-only">Toggle navigation</span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
        </button>
        <a class="navbar-brand"><span style="font-size: 32px !important; color: White;"><b>Amjad Khan</b></span></a>
      </div>
      <!-- Collect the nav links, forms, and other content for toggling -->
      <div class="collapse navbar-collapse" id="navbar">
        <ul class ="nav navbar-nav navbar-right">
          <li>
            <a href="https://mathbioinfo.github.io/amjadkhan/">Home</a>
          </li>
          <li>
            <a href="research.html">Research</a>
          </li>
	  <li>
           <a href="Data/cvkhan.pdf" target="_blank">CV</a>
          </li>
          <li>
            <a href="teaching.html">Teaching</a>
          </li>
          <li>
            <a href="presentations.html">Publications</a>
          </li>
	 <li>
            <a href="">Linux</a>
          </li>
		<li>
            <a href="bioinformatics.html">Bioinformatics</a>
          </li>
         </ul>
      </div><!-- /.navbar-collapse -->
    </div><!-- /.container -->
  </nav>
	
	
     <div class="container theme-showcase" role = "main">
       <div class="row"> 
               <div class="col-sm-2"></div> 
               <div class="col-sm-12">
			
	
		       
			<!--<div class="container theme-showcase" role="main"> -->
                 <div class="well my-well"> 
                    <div class="media">
                        <div class="col-sm-5 col-md-4 col-lg-3"> 
                            
	
		       

      </div>
  </div>
</div> 		
			
			

                    <div class="row bot-shrink20">
                        <div class="col-xs-9 vcenter">
                            <h3>Current ongoing projects</h3>
                        </div><!--
                        --><div class="col-xs-3 vcenter">
                       
                   </div> 
                </div>
                <div class="spacer10"></div>

                <hr>
			
				
			

                <div class="well my-well">
                    <div class="media">
                        <div class="col-sm-5 col-md-4 col-lg-3">
                            <br>
                            <center>
				 <img class="media-object left-shift" src="pics/Tn916.png" width=250/>
                                <img class="media-object left-shift" src="pics/ice.png" width=250/>
				    <br>
				    <br>
				 <img class="media-object left-shift" src="pics/phageome.png" width=250/>   
                            </center>
                            <br>
                        </div><!--
                        --><div class="col-sm-7 col-md-8 col-lg-9";>
                        <h4 class="media-heading" >
                            <strong >Antimicrobial Resistance: Emergence, Transmission, and Ecology</strong>
                        </h4>
                       <p style="text-align: justify;">
                            My interest in mathematical modelling, bacterial genome evolution, and bioinformatics led 
                            me to my present position as a postdoctoral fellow in the Faculty of Computer Science at 
                            Dalhousie University. Here, I am working with a group of bioinformaticians, computer scientists, 
                            and microbiologists on the Genome Canada- funded project 
                            <a href="https://www.genomecanada.ca/en/antimicrobial-resistance-emergence-transmission-and-ecology-arete" target="_blank">
                           "Antimicrobial Resistance: Emergence, Transmission, and Ecology (ARETE)". </a>
                            The ARETE project, comprised of scientists from Dalhousie University, Simon Fraser University, McMaster University, 
                            Agriculture and Agri-Food Canada (AAFC), and the Public Health Agency of Canada (PHAC), is looking for a better 
                            understanding of which genes contribute to antibiotic resistance and how they spread in the environment. 
                            I am applying my mathematical-modeling expertise to several projects in ARETE. The collaboration with government 
                            agencies provides the ARETE project with thousands of genomes of pathogenic species with relevance to agriculture 
                            and human disease, which creates unique opportunities and challenges from a modelling and bioinformatics point of view.
                       </p>
                        <P  style="text-align: justify";>
                            In addition to modelling this whole phenomenon to understand the spread of antimicrobial resistance (AMR) genes through the community and suggest some public policy 
	                    guidelines to curb or slow AMR spread, I am also interested in exploring the role of individual mobile genetic elements (MGEs), like integrative conjugative elements 
	                    (ICEs) and prophages, in the spread of these AMR genes. To accomplish this, I am using ODEs, PDEs, individual-based models, and bioinformatics tools. Bioinformatics 
	                    tools are used for the detection of MGEs (ICEs and prophages) and AMR genes in the bacterial genomes.  </p>
			    
			    <hr>
			    <ul>
				     
  				<li><p style="text-align: justify;"><a href="https://uwaterloo.ca/canadian-applied-industrial-math-society-annual-meeting-2021/" target="_blank">
					<strong> The role of temperate bacteriophages in the maintenance and distribution of Antibiotic Resistance Genes (ARGs).
					</strong></a> Amjad Khan, Lindi M. Wahl and Robert G Beiko.<em> Canadian Society of Applied and Industrial Mathematics (CAIMS 2021) </em> 
					June 21- 24, 2021 Virtual (hosted by the University of Waterloo, Waterloo, ON, Canada).</p> </li>
  				<li><p style="text-align: justify;"><a href="https://www.smb2021.org/" target="_blank">
					<strong> Modeling the transmission and loss of an important class of mobile genetic elements.
					</strong></a> Amjad Khan, Robert G Beiko.<em> Virtual annual Meeting and Conference of the Society for Mathematical Biology </em> 
					SMB 2021 June 13 -17.</p> </li>
				<li><p style="text-align:justify;">
					<a href="https://www.biorxiv.org/content/10.1101/2022.04.11.487771v1" target="_blank">
					<strong> Exploring the mobilome and resistome of Enterococcus faecium in a One Health context across two continents.
					</strong></a> Haley Sanderson, Kristen L. Gray, Alexander Manuele, Finlay Maguire, Amjad Khan ... Robert G. Beiko.<em> preprint,</em> 2022. </p> </li>
				      
 			    </ul>  
            </div>
        </div>
    </div>
			
	         <h3>Past projects</h3>

                <hr>

                <div class="well my-well">
                    <div class="media">
                        <div class="col-sm-5 col-md-4 col-lg-3">
                            <br>
                            <center>
				   <!-- <img class="media-object left-shift" src="pics/plosmodel.PNG" width=250/>-->
				   <!-- <br> -->
				    <!--<br> -->
                                <img class="media-object left-shift" src="pics/plos.PNG" width=250/>
                                <img class="media-object left-shift" src="pics/ploscomp.PNG" width=250/>
	                            <br>
				    <br>
				    <img class="media-object left-shift" src="pics/prophage.png" width=250/>
                            </center>
                            <br>
                        </div><!--
                        --><div class="col-sm-7 col-md-8 col-lg-9";>
                        <h4 class="media-heading">
                           <strong>The evolution and genetic repertoire of prophages (viral genomes integrated within a host bacterial genome) </strong>
                        </h4>
                        <p style="text-align: justify;">
			The evolution and gene content of viral genomes incorporated inside a host bacterial genome (prophages) were studied. 
				We created a mathematical model of the evolutionary dynamics that influence prophage size distribution and fit it 
				to three independent data sets. We were able to offer quantitative estimates of the relative rates of lysogeny, induction, 
				mutational degradation, and selection. We used bioinformatics approaches, analytical modelling, and computational simulation 
				to investigate the genetic repertoire of prophages. Our mathematical and computational approaches predicted that genes involved 
				in phage lytic function are preferentially lost, resulting in shorter prophages that often retain genes that benefit the host. 
				Informed by these models, we offered novel hypotheses for the enrichment of integrase and transposase genes in cryptic prophages. 
				Overall, we demonstrate that functional and cryptic prophages represent a diversity of genetic sequences that evolve along a 
				parasitism-mutualism continuum. </p>
			    <hr>
			    	<ul>
				     
  				<li><p style="text-align: justify;"><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008482" target="_blank">
				<strong> Evolution along the parasitism-mutualism continuum determines the genetic repertoire of prophages.
				</strong></a> Amjad Khan, Alita R. Burmeister, Lindi M. Wahl.<em> PLoS Computational Biology,</em> 2020.</p> </li>
  				<li><p style="text-align: justify;"><a href="https://www.sciencedirect.com/science/article/pii/S0040580919301844" target="_blank">
				<strong> Quantifying the forces that maintain prophages in bacterial genomes.
				</strong></a> Amjad Khan, Lindi M. Wahl.
				<em> Theoretical Population Biology,</em> 2020.</p> </li>				      
 			       </ul>  
			    
                            
            </div>
        </div>

</div>
		<div class="well my-well">
                    <div class="media">
                        <div class="col-sm-5 col-md-4 col-lg-3">
                            <br>
                            <center>
                                <img class="media-object left-shift" src="pics/phagantibio.png" width=250/>
                            </center>
                            <br>
                        </div><!--
                        --><div class="col-sm-7 col-md-8 col-lg-9";>
                        <h4 class="media-heading">
                        <strong>Synergistic action of phage and antibiotics</strong>
                        </h4>
                        <p style="text-align: justify;">
				The synergistic effect of phage therapy and antibiotics in the eradication of biofilm bacteria was investigated. 
				In this context, the biofilm was considered as a group defense mechanism, such that the functional response of phages 
				to the biofilm bacterial density reduces as the biofilm approaches carrying capacity. To capture this mechanism we 
				introdcued the function $f(x)=(κ−\frac{x}{K})x,$ where $x$ is the biofilm density, $K$ is the biofilm carrying capacity and 
				$1<κ<2$ is the group defense parameter. We predicted that complete elimination of biofilm bacteria can be achieved by 
				mechanisms that block the attachment of planktonic bacteria to the biofilm.
			</p>
			       <hr>
			    	<ul>
				     
  				<li><p style="text-align: justify;"><a href="https://link.springer.com/chapter/10.1007%2F978-3-319-99719-3_34" target="_blank">
				<strong> Phage therapy and antibiotics for biofilm eradication: a predictive model.
				</strong></a> Amjad Khan, Lindi M. Wahl, Pei Yu.
				<em> Recent Advances in Mathematical and Statistical Methods,</em> AMMCS 2017. </p> </li>  					      
 			       </ul>  
                            
            </div>
        </div>
</div>
		<div class="well my-well">
                    <div class="media">
                        <div class="col-sm-5 col-md-4 col-lg-3">
                            <br>
                            <center>
                                <img class="media-object left-shift" src="pics/fpu.png" width=250/>
                            </center>
                            <br>
                        </div><!--
                        --><div class="col-sm-7 col-md-8 col-lg-9";>
                        <h4 class="media-heading">
                       <strong>Approximations of lattice dynamics</strong>
                        </h4>
                        <p style="text-align: justify;">
				This study was devoted to the study of the Fermi-Pasta-Ulam (FPU) lattice dynamics. 
				FPU lattice with the nonlinear potential which leads to the generalized Korteweg-de Vries (gKDV) equation. 
				We showed that the $H^s (\mathbb{R})$ norm of the solution of the gKDV equation is bounded by a time-independent constant 
				in the subcritical case, whereas the $H^s (\mathbb{R})$ norm grows at most exponentially in time in the critical and supercritical cases. 
				With the help of these results, we extended the time scale for the approximation of the traveling waves of the FPU lattice by the traveling waves of the gKDV equation logarithmically in the subcritical case.
			</p>
			    	<hr>
			    	<ul>
				     
  				<li><p style="text-align: justify;"><a href="https://arxiv.org/abs/1603.01289" target="_blank">
				<strong> Long-time stability of small FPU solitary waves.
				</strong></a> Amjad Khan, Dmitry Pelinovsky.<em> Discrete & Continuous Dynamical Systems,</em> 2017.  </p> </li>  					      
 			       </ul>  
                            
            </div>
         </div>
      </div>		
    </div>
</div>
<!--</div> -->
        <!--   </div> -->
   <!-- </div> -->
   
			
<hr>

<!-- Footer -->
<footer>
    <div class="media">
        <div class="col-xs-6">
            <div class="pull-left">
                <b>MathBioinfo</b> <a href="mailto:mathbioinfo@gmail.com"
                target="_blank">(mathbioinfo@gmail.com)</a>
            </div>
        </div>
        <div class="col-xs-6">
            <div class="pull-right">
                <b>Last Update:</b>
                <script language="JavaScript">
                document.write(document.lastModified);
                </script>
            </div>
        </div>
    </div>
</footer>
</div>

<!-- jQuery -->
<script src="js/jquery.js"></script>

<!-- Bootstrap Core JavaScript -->
<script src="js/bootstrap.js"></script>

</body>

</html>
